src/modules/dashboard/components/ColonyTotalFunds/ColonyTotalFunds.tsx

```diff
@@ -109,13 +109,13 @@ const ColonyTotalFunds = ({
       </div>
       <div className={styles.totalBalanceCopy}>
         <FormattedMessage {...MSG.totalBalance} />
-        <Button
-          appearance={{ theme: 'blue', size: 'small' }}
-          onClick={handleManageFunds}
+        <Link
```

Comment:
I think should be the `NavLink` component rather then just the `Link` one.

But I'm not sure, since both kinda do similar things. Maybe we should remove one of them...?